src/lib/ngsi/ContextElement.cpp

```diff
@@ -113,14 +113,17 @@ std::string ContextElement::render(ConnectionInfo* ciP, RequestType requestType,
 *
 * ContextElement::toJson - 
 */
-std::string ContextElement::toJson(RenderFormat renderFormat, const std::vector<std::string>& attrsFilter)
+std::string ContextElement::toJson(RenderFormat renderFormat, const std::vector<std::string>& attrsFilter) const
 {
   std::string out;
 
   if (renderFormat != NGSI_V2_VALUES)
   {
     out += entityId.toJson();
-    out += ",";
+    if (contextAttributeVector.size() != 0)
```

Comment:
Yeah, I got this unconditional comma always when using only ContextElement in the payload.
Had to change this.
Funny that NO OTHER toJson suffers from this until now ...